.env.example

```
@@ -6,6 +6,12 @@ DIRECT_URL=postgresql://postgres:postgres@localhost:5433/dig_dev
 # Supabase (本番環境)
 SUPABASE_URL=
 SUPABASE_KEY=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The SUPABASE_KEY key should go before the SUPABASE_URL key

(UnorderedKey)